test/e2e/e2e_report_test.go

```diff
@@ -81,12 +81,12 @@ apiVersion: "config.istio.io/v1alpha2"
 kind: rule
 metadata:
   name: rule1
-  namespace: istio-config-default
+  namespace: istio-system
 spec:
   selector: target.name == "*"
   actions:
-  - handler: fakeHandlerConfig.fakeHandler.istio-config-default
-    instances: [ reportInstance.samplereport.istio-config-default ]
+  - handler: fakeHandlerConfig.fakeHandler.istio-system
+    instances: [ reportInstance.samplereport.istio-system ]
```

Comment:
@mandarjog: while we are changing this, should we adopt the style of:

```
instances:
- reportInstance...
```
instead?

Reply:
good point. yes, we should.

@jmuk 
1. use the extended yaml format.
2. elide namespace from handler and instance names.

Reply:
done here. Regarding 2, do we want to elide namespaces in other parts, especially in `testdata/config` files?